### DISCUSSION TITLE
Custom/dedicated server in server.json

### DISCUSSION THREAD

**Discussion by iLostAllCTRL** [qdm12/gluetun#2530] [Q&A]
I am trying to add a dedicated IP address server from one of the supported providers (Pure) but I noticed that every time the container starts, it merges in the servers.json file and overwrites the server I am trying to add. Can I get around this and connect to my dedicated server's IP?
I am using Pure VPN with port forwarding so that the open port number is static.  The DNS for the dedicated server is not listed in the servers.json.
Setting the location to read only throws errors that stop the container from starting.

**Comment by executed** [comment]
Why is this closed? Very hot topic IMHO
